sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -724,6 +724,16 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
       messageParameters = Map("className" -> className, "operator" -> operator))
   }
 
+  def nonTimeWindowNotSupportedInStreamingError(
+      windowFuncs: String,
```

Comment:
Can the argument contain multiple names? If not, please, change it to `windowFunc`